lib/log.js

```diff
@@ -1,31 +1,100 @@
 "use strict";
 
 var chalk = require("chalk");
+var args = require("./args");
 
 // Wrap "type".
 var wrapType = function (type) {
   return "[builder" + (type ? ":" + type : "") + "]";
 };
 
+// Levels
+var LEVELS = {
+  info: 0,
+  warn: 1,
+  error: 2,
+  none: 3
+};
+
 /**
  * A super-small logger.
  */
-module.exports = {
-  // TODO(6): Configurable log levels.
-  // https://github.com/FormidableLabs/builder/issues/6
-  _logger: function () {
-    return console;
+var log = module.exports = {
+  /**
+   * Set log level from command flags or environment.
+   *
+   * **NOTE - Side Effects**: Mutates environment.
+   *
+   * @param {Object} opts         Options object
+   * @param {Object} opts.argv    Arguments array.
+   * @param {Object} opts.env     Environment object to mutate (Default `process.env`)
+   * @returns {void}
+   */
+  setLevel: function (opts) {
+    opts = opts || {};
+    var env = opts.env && opts.env.env || process.env;
+
+    // Try to determine log level from environment.
+    var level = env._BUILDER_ARGS_LOG_LEVEL;
+
+    // If not, determine log level from command line.
```

Comment:
As always, if it were me I'd pull this into a `getLevel` method.

```
var getLevel = function getLevel(level) {
  if (level) { return level; }
  
  var parsed = args.general(opts.argv);
  return parsed.quiet === true ? "none" : parsed.logLevel;
}
```

You could also make it a class method if you'd prefer.